packages/styles/src/components/list.scss

```diff
@@ -0,0 +1,54 @@
+@use './../themes/bootstrap/core' as *;
+@use '../tokens/components' as components;
+@use '../functions/tokens' as tokens;
+@use './../functions/icons';
+
+tokens.$default-map: components.$post-list;
+
+$sizing-icon: tokens.get('post-list-checks', 'sizing-icon');
+$padding-icon: tokens.get('post-list-checks', 'spacing-padding-icon');
+
+.list-checks {
+  list-style: none;
+  padding-left: 0;
+  padding-block: tokens.get('post-list-checks', 'spacing-padding-block-outer');
+  margin-bottom: 0;
+
+  > li {
+    position: relative;
+    padding-inline-start: calc(
+      #{tokens.get('post-list-checks', 'spacing-gap-inline-text'),
+      } + #{$sizing-icon}
+    );
+
+    &::before {
+      content: '';
+      position: absolute;
+      left: 0;
```

Comment:
The check is not centrally aligned with the first line of text of the list item
<img width="680" alt="image" src="https://github.com/user-attachments/assets/548371e8-b16e-4926-b393-0f8782c39412">

Reply:
The size of icon is fixed.